src/Microsoft.Management.UI.Internal/ManagementList/Common/WpfHelp.cs

```diff
@@ -419,10 +419,7 @@ public static bool TryExecute(this RoutedCommand command, object parameter, IInp
         {
             ArgumentNullException.ThrowIfNull(templateParent);
 
-            if (string.IsNullOrEmpty(childName))
-            {
-                throw new ArgumentNullException("childName");
-            }
+            ArgumentException.ThrowIfNullOrEmpty(childName);
```

Comment:
Extra blank line 😄

Reply:
@CarloToso you and I both missed this one ...

Reply:
😅 (I fixed it in #19241)

Reply:
Thank you! 😄